src/stieltjes.jl

```diff
@@ -75,4 +74,173 @@ end
     D = T \ apply(*, log.(abs.(x .- x')), P)
     c = inv(2*kr.A)
     T[kr,:] * Diagonal(Vcat(2*convert(V,π)*c*log(c), 2c*D.diag.args[2]))
-end
\ No newline at end of file
+end
+
+#################################################
+# ∫f(x)g(x)(t-x)^a dx evaluation where f and g given in coefficients
+#################################################
+# recognize structure of W = P' * ((t .- x).^a .* P)
+const PowKernelIntegralPoint{T, V, D, F} = QuasiArrays.ApplyQuasiArray{T, 2, typeof(*), Tuple{QuasiArrays.QuasiAdjoint{T, Legendre{Float64}}, QuasiArrays.BroadcastQuasiArray{T, 2, typeof(*), Tuple{QuasiArrays.BroadcastQuasiArray{T, 1, typeof(^), Tuple{ContinuumArrays.AffineQuasiVector{T, V, Inclusion{V, D}, T}, F}}, Legendre{Float64}}}}} # method only works for Legendre for now, so require Legendre OPs.
```

Comment:
This is weird. What we want is `(t .- x) .^ a .* P` to return `P * PowKernelMatrix(...)` so that `P' * ((t .- x) .^ a .* P) == P'P * PowKernelMatrix(...) = Diagonal(...) * PowKernelMatrix(...)`

Reply:
To make this work we need to capture the broadcasted machinery. Something like this:
```julia
julia> const RPowKernelPointBroadcast{F<:Real} = Base.Broadcast.Broadcasted{<:Any,<:Any,typeof(^),<:Tuple{AffineQuasiVector,F}}
Base.Broadcast.Broadcasted{var"#s59",var"#s58",typeof(^),var"#s57"} where var"#s57"<:Tuple{AffineQuasiVector,F} where var"#s58" where var"#s59" where F<:Real

julia> Base.broadcasted(::LazyQuasiArrayStyle, ::typeof(*), K::RPowKernelPointBroadcast, P::Legendre) = "foo" # instead return P * PowKernelMatrix(...)

julia> (t .- x).^a .* P
"foo"
```

Reply:
Yeah, I knew something was still weird about the way this was set up. I actually have all of what you said there already working, just wasn't sure what structure we want. This should help me fix it with the next commit, thanks!